### ISSUE TITLE
Reference page is not reachable since at least 1 day

### ISSUE THREAD

**Issue by a-hahn** [canonical/microk8s.io#637]
## Summary
https://microk8s.io/docs/command-reference 
cannot be reached. Site is down

**Comment by a-hahn** [comment]
Seems nobody takes care of broken documentation site :(
![Screenshot from 2023-12-12 16-51-30](https://github.com/canonical/microk8s.io/assets/777741/4a0e8168-1dfc-4736-b597-e0f5d1e88c66)

**Comment by friendywill** [comment]
Top tip, use the wayback machine if you desperately need to access any site that is down, google wayback machine, enter in [https://microk8s.io/docs/command-reference](https://microk8s.io/docs/command-reference), then you are good.

**Comment by primoz-p** [comment]
It is still not working - https://microk8s.io/docs/command-reference.

**Comment by skatsaounis** [comment]
Hi @a-hahn and others

I was hitting the issue like you and tried to reproduce. It seems that a python dependency was missing at least a minor release, which includes the fix. I created a PR which hopefully brings command reference back to life. Let's wait for the project collaborators to take a look :)

To get you unblocked, the command reference is also accessible through [Kubernetes Discourse](https://discuss.kubernetes.io/t/command-reference/11268). Actually, microk8s.io is consuming Discourse to display the docs as I found out while playing with this repo.
